components/match2/commons/starcraft_starcraft2/match_group_input_starcraft.lua

```diff
@@ -862,7 +862,12 @@ function StarcraftMatchGroupInput.ProcessPlayerMapData(map, match, OppNumber)
 			for j = 1, 4 do
 				if not Logic.isEmpty(map['t' .. i .. 'p' .. j]) then
 					if map['t' .. i .. 'p' .. j] ~= 'TBD' and map['t' .. i .. 'p' .. j] ~= 'TBA' then
-						map['t' .. i .. 'p' .. j] = mw.ext.TeamLiquidIntegration.resolve_redirect(map['t' .. i .. 'p' .. j])
+						-- allows fetching the link of the player from preset wiki vars
+						-- map['t' .. i .. 'p' .. j .. 'link'] for manual overwrite
+						-- map['t' .. i .. 'p' .. j] for the input of the displayName
+						local mapPlayer = map['t' .. i .. 'p' .. j .. 'link'] or
+							Variables.varDefault(map['t' .. i .. 'p' .. j] .. '_page', map['t' .. i .. 'p' .. j])
+						map['t' .. i .. 'p' .. j] = mw.ext.TeamLiquidIntegration.resolve_redirect(mapPlayer)
```

Comment:
```suggestion
						local playerIndex = 't' .. i .. 'p' .. j
						local mapPlayer = map[playerIndex .. 'link'] or
							Variables.varDefault(map[playerIndex] .. '_page') or map[playerIndex]
						map[playerIndex] = mw.ext.TeamLiquidIntegration.resolve_redirect(mapPlayer)
```
* Extracted the repeated string to a variable
* took the last fallback out of the Variable. Due to way it was setup, it looked more like a Variable lookup

Reply:
tbh in the rewrite i want to make the var link retrieval a sep. function as it gets used at several places in this module

there is so much other stuff to clean up too
like variable names used in the module etc.